include/ini/simpleIniParser/ini.hpp

```diff
@@ -30,7 +30,9 @@ namespace simpleIniParser {
             ~Ini();
             std::string build();
             IniSection * findSection(std::string name);
+            IniSection * findOrCreateSection(IniSectionType type, std::string name);
```

Comment:
These functions should probably be added to https://github.com/AtlasNX/SimpleIniParser as well. Tbh I'm not sure why I didn't just just made a submodule out of this

Reply:
Waiting for [PR from SimpleIniParser](https://github.com/AtlasNX/SimpleIniParser/pull/2) now